Bots/JavaScript/Consumers/CodeFirst/SimpleHostBot/package.json

```diff
@@ -18,6 +18,7 @@
   "dependencies": {
     "botbuilder": "~4.11.0",
     "botbuilder-dialogs": "~4.11.0",
+    "botframework-connector": "~4.11.0",
```

Comment:
Do w e need to reference botframework-connector? I don't see that referenced in any of skill samples? ([here](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/javascript_nodejs/81.skills-skilldialog/dialogRootBot/package.json))

Reply:
We reviewed it and we can remove it, since it's not being used. We'll remove it now. Thanks!